disco_aws_automation/disco_alarm.py

```diff
@@ -10,6 +10,7 @@
 
 # Max batch size for alarm deletion http://goo.gl/vMQOrX
 DELETE_BATCH_SIZE = 100
+ENVIRONMENT_DELETE_SKIP_NAMESPACES = ['AWS/ES']
```

Comment:
Can we use config variable instead and make configurable? Based on namespace globally per alarm locally?